src/mono/wasm/runtime/pinvoke.h

```diff
@@ -23,4 +26,29 @@ wasm_dl_get_native_to_interp (const char *key, void *extra_arg);
 void
 mono_wasm_pinvoke_vararg_stub (void);
 
+
+typedef void* (*MonoWasmNativeToInterpCallback) (char * cookie);
+
+void
+mono_wasm_install_interp_to_native_callback (MonoWasmNativeToInterpCallback cb);
+
+typedef struct {
+	
+} InterpMethodArguments;
```

Comment:
1. The usual way to do this would be something like `typedef struct _InterpMethodArguments InterpMethodArguments` without providing a definition of `struct _InterpMethodArguments` in this header file.
2. Use a `Mono` prefix on the struct definition: `typedef struct _MonoInterpMethodArguments MonoInterpMethodArguments`